addons/medical_treatment/functions/fnc_bandage.sqf

```diff
@@ -21,7 +21,7 @@
  * Public: No
  */
 
-_this set [6, _this param [6, 1]]; // set default Bandage effectiveness coefficient
+_this set [6, 1]; // set bandage effectiveness coefficient - overwrites bool of createLitter
```

Comment:
https://github.com/acemod/ACE3/pull/9924/files added `_createLitter`, in which we forgot to take `_bandageEffectiveness` into account.

`_bandageEffectiveness` was added in ACE 3.17.0 to allow 3rd party mods to modify things more easily (see https://github.com/acemod/ACE3/pull/9814/files and https://github.com/acemod/ACE3/pull/9865). Given it's a recent change, breaking BWC for the sake for fixing 2 things at once it worth it imo.

```suggestion
_this set [7, _this param [7, 1]]; // set bandage effectiveness coefficient
```

I haven't tested the code above yet, I can test it in about 12 hours, so do not merge suggestion yet.
If we were to merge said code, we'd need to update the header as well.

Reply:
```suggestion
params ["_medic", "_patient", "_bodyPart", "_classname", "", "", "_bandageEffectiveness"];
params ["_medic", "_patient", "_bodyPart", "_classname", "", "", "_bandageEffectiveness"];
```

Reply:
```
params ["_medic", "_patient", "_bodyPart", "_classname", "", "", "_bandageEffectiveness"];
```
This will also need to become for the4 above to work
```
params ["_medic", "_patient", "_bodyPart", "_classname", "", "", "", "_bandageEffectiveness"];
```

Reply:
Yep. I had thought about it, was going to add it to the suggestion, then completely forgot xD

Reply:
Made the changes and tested